tests/runtime/test_interface.py

```diff
@@ -50,3 +53,16 @@ def test_interface_descriptor__to_dict(interface: Interface):
             },
             'out_type': {'field': 5, 'type': 'test_container'}
         }]}
+
+
+@pytest.fixture
+def pipeline():
+    return Pipeline('Test Pipeline', [PipelineStep('a', 'b'), PipelineStep('c', 'd')], NumpyNdarrayDatasetType,
+                    NumpyNdarrayDatasetType)
+
+
+def test_pipeline_interface(pipeline):
+    # Dunno if this really helps?
+    iface = pipeline_interface(pipeline)
+    assert 'run' in iface.exposed_methods()
+    assert str(iface.exposed_method_args('run')[0]) == 'data: NumpyNdarrayDatasetType'
```

Comment:
very strange assert)

Reply:
Well i was trying to check if argument passed correctly from input_data to exposed method "run"